karaf/jetty-config/src/main/resources/jetty.xml

```diff
@@ -56,6 +56,9 @@
                         <Item>org.apache.brooklyn.rest.security.jaas.BrooklynLoginModule$RolePrincipal</Item>
                     </Array>
                 </Set>
+                <Set name="identityService">
+                    <New class="org.eclipse.jetty.security.DefaultIdentityService"/>
+                </Set>
```

Comment:
@neykov But if two threads are invoking that code an `java.lang.IllegalStateException` is thrown